contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -224,12 +229,22 @@ class Meta:
         )
 
 
+class ThumbnailEncodingFieldsSerializer(JSONFieldDictSerializer):
+    type = DictField(allow_null=True)
```

Comment:
Is this a copypasta from the `ExtraFieldsSerializer`?

Reply:
A slightly modified but yes, copy and pasted. Good catch!